app/(main)/_layout.js

```diff
@@ -21,7 +21,7 @@ export default function Layout() {
 				/>
 
 				<ExpandableButton
-					btnStyles='absolute top-20 left-5'
+					btnStyles='absolute top-10 left-5'
```

Comment:
@dev-arekusandoru this needs to be discussed further. The current iteration of the Figma has them equally spaced both in the y and x from the borders of the screen. All previous iterations of our repo have incorrect.

Reply:
@aidan-hubley right. the issue here is not the figma, but rather how we achieve that in the code. Right now the _layout page renders the buttons 40 something pixels from the top of my screen which looks right. it looks different on your screen, however, because of the different safe area size. Like i mentioned in another comment, this is something that needs looking into regarding SafeArea spacing in the _layout.js file